developer/installation/installation-windows.md

```diff
@@ -0,0 +1,124 @@
+# Installation for Windows
+
+## Install prerequisites:
+
+**Note: These commands all assume you are running them from an administrator shell**
+
+### Install node
+Download and run the installer from the [NodeJs site](https://nodejs.org)
+
+### Install Chocolately
+
+Visit the [Chocolatey site](https://chocolatey.org/install) and follow the instructions
+
+
+### Install Git
+
+```sh
+choco install git
+````
+
+### Install Meteor
+```sh
+choco install meteor
+```
+
+### Install ImageMagick (Optional but recommended)
+
+```sh
+choco install imagemagick
+```
+
+### Install Windows Build Tools 2015
+
+```sh
+npm install -g windows-build-tools
+```
+
+
+## Install Reaction
+
+### Install the Reaction command line client
+
+```sh
+# install CLI
+npm install -g reaction-cli
+```
+
+### Create your local Reaction installation
+
+You should preferably create a directory under your user, do not run this from the `\Windows\system32` directory as you won't have the correct permissions
+
+```sh
+# clone Reaction, install NPM dependencies
+reaction init
+cd reaction
+```
+
+If you don't want Reaction to be installed in the default `reaction` directory you can
+specify a direction like
+```sh
+reaction init my-new-reaction-project
+cd init my-new-reaction-project
+```
+
+
+### Start Reaction
+
+To start Reaction, run the `reaction` command
+
+```sh
+# start Reaction
+reaction
+# or
+reaction run
+```
+
+**Note that the first run can take a while as it downloads dependencies. This is especially true if you are not in North America**
+
+_The initial admin user for the site is auto generated, and displayed in your console (or see: env variables section to default these)_
```

Comment:
Since the portion below this is the same for all 3 systems, I think we should separate that out and link it.

Reply:
I really wanted to avoid anybody having to leave the one page. I understand that it repeats but I think it's worth the repetition.

Reply:
I guess anything past the "Congratulations" part could be moved since it's post-install

Reply:
Okay, I moved all of that stuff about the CLI commands to a separate page